source/macos/AdaptiveCards/AdaptiveCards/Views/ACRView.swift

```diff
@@ -80,6 +82,8 @@ class ACRView: ACRColumnView {
             // If it is a textfield, the entered text gets selected when focus is set, so taking the cursor to the last position it was present
             guard let focusedTextField = focusedElementOnHideError as? ACRTextField else { return }
             focusedTextField.resetCursorPositionIfNeeded()
+        } else if firstFieldWithError != nil {
+            firstFieldWithError?.setFocus()
         }
```

Comment:
Why is this included inside keyboard reset flow?
This is something related to ScreenReaders only right?